wesnoth-ei.po

```diff
@@ -11782,29 +11829,37 @@ msgid ""
 "i></span>\n"
 "<b></b>"
 msgstr ""
+"<span size='small'><i> Tento předmět <b>nelze</b> upustit a <b>není</b> "
+"znovu použitelný, pokud jeho uživatel zemře.</i></span>\n"
+"<span size='small'><i> Sebrání předmětu spotřebuje pohyby a útok jednotky.</"
+"i></span>\n"
+"<b></b>"
 
 #: data/campaigns/Eastern_Invasion/utils/items.cfg:136
 msgid "quiver"
-msgstr ""
+msgstr "toulec"
 
 #: data/campaigns/Eastern_Invasion/utils/items.cfg:137
 msgid "Crystal Quiver"
-msgstr ""
+msgstr "Křišťálový toulec"
 
 #: data/campaigns/Eastern_Invasion/utils/items.cfg:138
 msgid ""
 "Arrows from this crystalline quiver glimmer with a pale magical light, "
 "<i><b>illuminating</b></i> the surrounding area and making your bow or "
 "crossbow attacks <i><b>arcane</b></i>."
 msgstr ""
+"Šípy z tohoto krystalového toulce se třpytí bledým magickým světlem, "
```

Comment:
```suggestion
"Šípy z tohoto křišťálového toulce se třpytí bledým magickým světlem, "
```

Reply:
No, přemýšlel jsem nad tím, jestli tam dát křišťálový, krystalový či krystalický. Přišlo mi příliš prvoplánové tam zopakovat křišťálový, ale budiž.

Reply:
Rozhodně nechci hasit tvé tvůrčí nadšení, jen, možná hloupě, předpokládám, že ten toulec je skleněný, ne přímo z nerostu.

Reply:
Bylo mi vysvětleno, že křišťálový toulec je křišťály vykládaný, tak jsem tam to krystalového vrátil.